models/ccsr/final/ccsr__singular_condition_category.sql

```diff
@@ -15,6 +15,10 @@ select
     {{ var('dxccsr_version') }} as dxccsr_version,
     '{{ var('tuva_last_run')}}' as tuva_last_run
 from {{ref('ccsr__long_condition_category')}}
-where 
-    is_{{ var('record_type', 'ip') }}_default_category = true
+where
+    {% if target.type == 'fabric' %}
+        is_{{ var('record_type', 'ip') }}_default_category = 1
+    {% else %}
+        is_{{ var('record_type', 'ip') }}_default_category = true
+    {% endif %}
```

Comment:
I wonder... if other DWs accept 1 instead of true (0 instead of false) we just use that syntax? Just trying to think through ways to simple the code and not have all of these Fabric exceptions.

Reply:
Updated the logic.